exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -108,42 +108,45 @@
 
   /**
    * Returns blockchain blocks information for the requested range. The blocks are returned
-   * in reverse order, starting from the {@code heightMax}.
-   * @param count Number of blocks to return.
-   *        It should be in range [1, {@linkplain ExonumApi#MAX_BLOCKS_PER_REQUEST}]
-   * @param blockFilter controls whether to skip blocks with no transactions
-   * @param heightMax maximum height of the returned blocks.
+   * in ascending order by their height.
+   *
+   * @param fromHeight the height of the first block to include. Must be non-negative
+   * @param toHeight the height of the last block to include. Must be greater than
+   *        or equal to {@code fromHeight} and less than or equal to the blockchain height.
    *        If the {@code heightMax} is greater than actual blockchain height then
```

Comment:
old param name in the doc.

Reply:
Clarified that such behaviour will be fixed in subsequent release (though it was tempting to fix it now with an additional request)